docs/alerts/newalert.md

```diff
@@ -54,6 +54,24 @@ Channels can be created once and reused across alerts.
 
 Frequency and time interval between notifications can be set in Email Alert Frequency section. Click **Next**.
 
+-   SNS
+
+You can configure alerts to send to an AWS SNS topic. To receive alerts on an SNS topic, you need to configure the SNS topic.
+
+![Create email channel](./images/newalert/4-alert-sns.png)
+
+1.  Select SNS as the communication channel.
+    
+2.  Enter details of your SNS topic, including SNS topic Arn: An AWS resource identifier, AWS access key id, and secret key which has access publish to this topic.
```

Comment:
Should "Arn" be capitalized (i.e. ARN)?

Should also read, "...which has access to publish to this topic."